app/playbooks/deploy.d/container-storage/local-storage.yml

```diff
@@ -5,7 +5,7 @@
 
   tasks:
     - name: get storage node drive id paths
```

Comment:
we could hand this out to the LSO Operator nowadays... it can scan nodes and add their disks as PVs that we can consume.

Then in the ocs-storagecluster.yml.j2 we can count those PVs (they will have a Storage Class name that we define)

Reply:
Good call. That was not available when we started. I've opened an issue #165 to update this in the future. It is a worthy cause, but probably too much change for this one PR.